components/logins/src/db.rs

```diff
@@ -419,6 +419,82 @@ impl LoginDb {
         Ok(login)
     }
 
+    pub fn import_multiple(&self, logins: &[Login]) -> Result<u64> {
+        let tx = self.unchecked_transaction()?;
+        let now_ms = util::system_time_ms_i64(SystemTime::now());
+        let sql = format!(
+            "INSERT OR IGNORE INTO loginsL (
+                hostname,
+                httpRealm,
+                formSubmitURL,
+                usernameField,
+                passwordField,
+                timesUsed,
+                username,
+                password,
+                guid,
+                timeCreated,
+                timeLastUsed,
+                timePasswordChanged,
+                local_modified,
+                is_deleted,
+                sync_status
+            ) VALUES (
+                :hostname,
+                :http_realm,
+                :form_submit_url,
+                :username_field,
+                :password_field,
+                :times_used,
+                :username,
+                :password,
+                :guid,
+                :time_created,
+                :time_last_used,
+                :time_password_changed,
+                :local_modified,
+                0, -- is_deleted
+                {new} -- sync_status
+            )",
+            new = SyncStatus::New as u8
```

Comment:
Really we want SyncStatus::Unknown (#648) for this... But I think this isn't wrong in a way we're not already wrong elsewhere.